### DISCUSSION TITLE
Led K4027C/L4027C/N4027C/NT4027C

### DISCUSSION THREAD

**Discussion by developerGM** [Koenkk/zigbee2mqtt#18899] [Devices]
How can I change red led?
Using BTicino original app mobile this led I can set in blue color.
Using zigbee2mqtt I see only red
I am testing "led in dark" on or off but the result not change....always fixed red

**Comment by mattibal** [comment]
I think the red LED in the device means something like "error". 
I've seen the red light usually appears when you have just paired the device to Zigbee2mqtt. It disappears after you have rebooted the device several times (by switching power off and on) or reassociated to Zigbee2mqtt.
While there is the red light the device doesn't work correctly, if I remember well it doesn't let you move the shutter in a middle position, it just goes all the way up or down.
